pkg/nmstate/policy.go

```diff
@@ -320,6 +321,77 @@ func (builder *PolicyBuilder) WithVlanInterface(baseInterface string, vlanID uin
 	return builder.withInterface(newInterface)
 }
 
+// WithVlanInterfaceIP adds VLAN interface with IP configuration to the NodeNetworkConfigurationPolicy.
+func (builder *PolicyBuilder) WithVlanInterfaceIP(baseInterface string, vlanID uint16,
```

Comment:
Function's argument of the same type should be placed next to each other.

Reply:
Hi Nikita fixed your comment. Thanks